includes/blocks/safe-svg/edit.js

```diff
@@ -155,47 +174,34 @@ const SafeSvgBlockEdit = ( props ) => {
 							onError={onError} />
 					</BlockControls></>
 			}
-			<MediaUpload
-				onSelect={onSelectImage}
-				allowedTypes={ALLOWED_MEDIA_TYPES}
-				accept={ALLOWED_MEDIA_TYPES}
-				value={imageID}
-				render={({open}) => {
-					return (
-						<div
-							style={{
-								maxWidth: '100%',
-								textAlign: alignment
-							}}
-						>
-							{!svgURL &&
-								<Button variant="tertiary" onClick={open}>
-									{__('Select an SVG icon', 'safe-svg')}
-								</Button>
-							}
-							{svgURL &&
-								<svg
-									style={{
-										width: dimensionWidth,
-										height: dimensionHeight,
-										maxWidth: '100%',
-										maxHeight: '100%'
-									}}
-								>
-									<image
-										xlinkHref={svgURL}
-										src={svgURL}
-										width={dimensionWidth < dimensionHeight ? dimensionWidth : '100%'}
-										style={{
-											height: dimensionWidth > dimensionHeight ? dimensionHeight : 'auto'
-										}}
-									/>
-								</svg>
-							}
-						</div>
-					);
-				}}
-			/>
+
+
+			{!svgURL &&
+				<MediaPlaceholder
+					onSelect={onSelectImage}
+					allowedTypes = {ALLOWED_MEDIA_TYPES}
+					accept={ALLOWED_MEDIA_TYPES}
+					value={imageID}
+					labels={{
+						title: __( 'Inline SVG', 'safe-svg' ),
+						instructions: __( 'Upload an SVG or pick one from your media library.', 'safe-svg' )
+					}}
+				/>
+			}
+
+			{svgURL &&
+				<div className={newClassName} style={{ textAlign: alignment }}>
+					<div
+						{ ...containerBlockProps }
```

Comment:
Thanks for making the changes @bmarshall511. I think we should move this to the root `<div>` element. Otherwise, the user will not be able to select a block by clicking on it.

Reply:
@iamdharmesh Ah, nice catch! Sorry about that. I've updated it accordingly.